server/src-lib/Hasura/RQL/DDL/EventTrigger.hs

```diff
@@ -307,6 +309,24 @@ runInvokeEventTrigger (InvokeEventTriggerQuery name payload) = do
       Just True -> return ()
       _         -> throw400 NotSupported "manual mode is not enabled for event trigger"
 
+runPauseEventTrigger
+  :: (MonadTx m)
+  => PauseEventTriggerQuery -> m EncJSON
+runPauseEventTrigger (PauseEventTriggerQuery name) = do
+  affectedRows <- updatePauseOfEventTriggerInCatalog name True
+  if | affectedRows == 1 -> pure successMsg
+     | affectedRows == 0 -> throw400 NotFound "event trigger not found"
+     | otherwise -> throw500 "unexpected: more than one event triggers have been paused"
```

Comment:
This is a weird quirk of the English language that doesn’t make any sense, but the phrase “more than one *x*” [is a grammatically *singular* subject](https://ell.stackexchange.com/a/109365/4507). That means the correct wording of this error message is actually:

> unexpected: more than one event trigger has been paused

Why? I have no idea.

That said, I’d actually reword this error message more significantly, since I think as-written it is confusing. If I got this error message, I would expect that more than one of my event triggers actually got paused, but in reality, the transaction will be rolled back, so *no* event triggers will have been paused. Therefore, I’d word this a different way:

```suggestion
     | otherwise -> throw500 "multiple event triggers exist with the given name"
```